infrastructure/stacks/test-stack-2/s3.tf

```diff
@@ -1,4 +1,4 @@
 module "test_s3_bucket" {
   source      = "../../modules/s3"
-  bucket_name = "test-s3-bucket-2"
+  bucket_name = "nhse-uec-dos-mgm-test-s3-bucket-2"
```

Comment:
sorry I am sure it is me being thick but how does this make it more unique than it was before ?

Reply:
haha, no prob. Basically S3 buckets have to be **globally** unique (well unique to an AWS segment anyway). I think someone else thats using AWS has a test-s3-bucket already. So prefixing with nhse-uec-dos-mgm will help